uber/templates/emails/shifts/reminder.txt

```diff
@@ -4,7 +4,7 @@ Thanks again for signing up to volunteer at {{ c.EVENT_NAME }}!  You're not curr
 
 Please sign up for shifts at {{ c.URL_BASE }}/signups/login?first_name={{ attendee.first_name|urlencode }}&last_name={{ attendee.last_name|urlencode }}&email={{ attendee.email|urlencode }}&zip_code={{ attendee.zip_code|urlencode }} -- if you need to, you can verify/update your personal information at {{ c.URL_BASE }}/preregistration/confirm?id={{ attendee.id }}
 
-You can add / drop / change your shifts anytime until {{ c.FINAL_EMAIL_DEADLINE|datetime_local }}.
+You can add / drop / change your shifts anytime until {{ c.FINAL_EMAIL_DEADLINE|datetime_local }}. Counts will be taken for perks by the end of December 23rd.
```

Comment:
Normally this would be a config option, but we want this change Right Now so we'll use text for now.